src/Jira/Api.php

```diff
@@ -741,16 +741,15 @@ public function api(
 				}
 			}
 
-			if ( $return_as_array ) {
+			if ($return_as_array) {
```

Comment:
Please restore spaces, that were removed.

---

As you can see from majority of above changed IFs the condition is wrapped in spaces.